pluginconfig/commit_test.go

```diff
@@ -111,124 +111,124 @@ func TestArbitrumPriceSource_Validate(t *testing.T) {
-// 				TokenInfo: map[types.Account]TokenInfo{
-// 					remoteTokenAddress: {
-// 						AggregatorAddress: aggregatorAddress,
-// 						DeviationPPB:      cciptypes.BigInt{Int: big.NewInt(1)},
-// 						Decimals:          18,
-// 					},
-// 				},
-// 				NewMsgScanBatchSize:                256,
-// 				MaxReportTransmissionCheckAttempts: 10,
-// 				MaxMerkleTreeSize:                  1000,
-// 			},
-// 			true,
-// 		},
-// 		{
-// 			"invalid, price sources with no chain selector",
-// 			fields{
-// 				RemoteGasPriceBatchWriteFrequency: *commonconfig.MustNewDuration(1),
-// 				TokenPriceBatchWriteFrequency:     *commonconfig.MustNewDuration(1),
-// 				TokenInfo: map[types.Account]TokenInfo{
-// 					remoteTokenAddress: {
-// 						AggregatorAddress: aggregatorAddress,
-// 						DeviationPPB:      cciptypes.BigInt{Int: big.NewInt(1)},
-// 						Decimals:          18,
-// 					},
-// 				},
-// 				NewMsgScanBatchSize:                256,
-// 				MaxReportTransmissionCheckAttempts: 10,
-// 				MaxMerkleTreeSize:                  1000,
-// 			},
-// 			true,
-// 		},
-// 		{
-// 			"invalid, no new msg scan batch size",
-// 			fields{
-// 				RemoteGasPriceBatchWriteFrequency:  *commonconfig.MustNewDuration(1),
-// 				TokenPriceBatchWriteFrequency:      *commonconfig.MustNewDuration(1),
-// 				TokenInfo:                          map[types.Account]TokenInfo{},
-// 				MaxReportTransmissionCheckAttempts: 10,
-// 				MaxMerkleTreeSize:                  1000,
-// 			},
-// 			true,
-// 		},
-// 	}
-// 	for _, tt := range tests {
-// 		t.Run(tt.name, func(t *testing.T) {
-// 			c := CommitOffchainConfig{
-// 				RemoteGasPriceBatchWriteFrequency:  tt.fields.RemoteGasPriceBatchWriteFrequency,
-// 				TokenPriceBatchWriteFrequency:      tt.fields.TokenPriceBatchWriteFrequency,
-// 				TokenInfo:                          tt.fields.TokenInfo,
-// 				PriceFeedChainSelector:             tt.fields.TokenPriceChainSelector,
-// 				NewMsgScanBatchSize:                int(tt.fields.NewMsgScanBatchSize),
-// 				MaxReportTransmissionCheckAttempts: uint(tt.fields.MaxReportTransmissionCheckAttempts),
-// 				MaxMerkleTreeSize:                  uint64(tt.fields.MaxMerkleTreeSize),
-// 			}
-// 			err := c.Validate()
-// 			if tt.wantErr {
-// 				require.Error(t, err)
-// 			} else {
-// 				require.NoError(t, err)
-// 			}
-// 		})
-// 	}
-// }
+func TestCommitOffchainConfig_Validate(t *testing.T) {
+	type fields struct {
+		RemoteGasPriceBatchWriteFrequency  commonconfig.Duration
+		TokenPriceBatchWriteFrequency      commonconfig.Duration
+		TokenInfo                          map[types.Account]TokenInfo
+		TokenPriceChainSelector            cciptypes.ChainSelector
+		TokenDecimals                      map[types.Account]uint8
+		NewMsgScanBatchSize                uint32
+		MaxReportTransmissionCheckAttempts uint32
+		MaxMerkleTreeSize                  uint32
+	}
+	//nolint:gosec
+	const remoteTokenAddress = "0x260fAB5e97758BaB75C1216873Ec4F88C11E57E3"
```

Comment:
How about using randomized values? (or at least making it a regular variable and get rid of this linter ignore) 

report_test.go contains
```go
func randomAddress() string {
	b := make([]byte, 20)
	_, _ = crand.Read(b) // Assignment for errcheck. Only used in tests so we can ignore.
	return cciptypes.Bytes(b).String()
}
```

Maybe we could just move this function to some place like testhelpers/common.go and reuse whenever we need Eth like addresses in tests and have

```go
remoteTokenAddress := testhelpers.RandomAddress()
aggregatorAddress := testhelpers.RandomAddress()
```